### PR TITLE
Fixed links on audit home page

### PR DESCRIPTION
**Pull request by samdutton**
Corrected links to ILT content; added `{: .external}` missing from a couple of external links.

**CC:** @petele

### CODE REVIEW

**Comment by samdutton** [comment]
Typo warnings are not relevant here.

Line 89 cannot be reduced in length.

File: src/content/en/fundamentals/performance/audit/index.md [hunk 1]
```diff
@@ -81,20 +85,24 @@ If nothing else, simply use [Lighthouse](/web/tools/lighthouse/) to check for:
 
   * HTTPS: [every site should deliver all assets over HTTPS](/web/fundamentals/security/encrypt-in-transit/why-https).
   * Server settings: your web server or CDN should [use compression correctly](/web/fundamentals/performance/optimizing-content-efficiency/optimize-encoding-and-transfer), [use HTTP/2](/web/fundamentals/performance/http2/), and [include appropriate headers](/web/fundamentals/performance/optimizing-content-efficiency/http-caching) to enable your browser to cache resources.
-  * Script elements that can be moved to the bottom of the page and/or given an [async or defer](http://peter.sh/experiments/asynchronous-and-deferred-javascript-execution-explained/) attribute.
+  * Script elements that can be moved to the bottom of the page and/or given 
+an [async or defer](http://peter.sh/experiments/asynchronous-and-deferred-javascript-execution-explained/){: .external} attribute.
```
Comment (petele) [inline src/content/en/fundamentals/performance/audit/index.md]: lines should wrap at 80 chars

File: src/content/en/fundamentals/performance/audit/index.md [hunk 2]
```diff
@@ -81,20 +85,24 @@ If nothing else, simply use [Lighthouse](/web/tools/lighthouse/) to check for:
...
   * JavaScript and libraries that can be removed.
-  * [Unused CSS](https://umaar.com/dev-tips/121-css-coverage/) and [unused JavaScript](/web/updates/2017/04/devtools-release-notes).
+  * [Unused CSS](https://umaar.com/dev-tips/121-css-coverage/){: .external} 
+and [unused JavaScript](/web/updates/2017/04/devtools-release-notes).
```
Comment (petele) [inline src/content/en/fundamentals/performance/audit/index.md]: You should indent this line so it's inline with the previous

File: src/content/en/fundamentals/performance/audit/index.md [hunk 3]
```diff
@@ -81,20 +85,24 @@ If nothing else, simply use [Lighthouse](/web/tools/lighthouse/) to check for:
...
   * Images that can be saved with higher compression or smaller pixel dimensions.
-  * Image files that would be smaller saved using a different format, for example photos saved as PNGs.
+  * Image files that would be smaller saved using a different format, for example 
+photos saved as PNGs.
```
Comment (petele) [inline src/content/en/fundamentals/performance/audit/index.md]: indentation

File: src/content/en/fundamentals/performance/audit/index.md [hunk 4]
```diff
@@ -81,20 +85,24 @@ If nothing else, simply use [Lighthouse](/web/tools/lighthouse/) to check for:
...
 
-Priorities for refactoring depend on your audience, content and functionality. Who visits your site?
-Why and how do they use it? What's your
-[performance budget](https://www.performancebudget.io/){:.external}? If you're not sure of the answer
-to these questions, try the requirements gathering exercises from our PWA training resources:
-[Your audience, your content](https://docs.google.com/presentation/d/1-UjSS-kRZgE0q77zPBkgjg2-huXF2GobzhSqwI0AdU8/edit#slide=id.g16e897f04a_0_14){: .external} and [Design for all your users](https://docs.google.com/presentation/d/14CaiTF3f-OFoWs84lXeurE95_LJIFJwC-xhMky62dAg/edit#slide=id.g16ea125a27_0_6){: .external}.
+Priorities for refactoring depend on your audience, content and functionality. 
+Who visits your site? Why and how do they use it? What's your
+[performance budget](https://www.performancebudget.io/){:.external}? If you're not sure of the 
+}answer to these questions, try the requirements gathering exercises from our PWA training
```
Comment (petele) [inline src/content/en/fundamentals/performance/audit/index.md]: errant `}` at start of line.

File: src/content/en/fundamentals/performance/audit/index.md [hunk 5]
```diff
@@ -58,14 +58,18 @@ against peers and estimate the potential revenue opportunity of improving site s
...
+runs performance tests from multiple locations globally using real browsers with real connections, 
+to provide detailed metrics and optimization advice.
   <ul>
-    <li>Leverage <a href="(https://www.webpagetest.org/easy">webpagetest.org/easy</a> to quickly simulate differences between network connection types on mobile</li>
+    <li>Leverage <a href="(https://www.webpagetest.org/easy">webpagetest.org/easy</a> to quickly 
+simulate differences between network connection types on mobile</li>
     <li>Generate a Lighthouse audit and report with every WebPagetest trace.</li>
-    <li> Measure site on first visit as well as for repeat views — for example: to see how much service worker caching helps.</li>
+    <li> Measure site on first visit as well as for repeat views — for example: to see how much 
+service worker caching helps.</li>
```
Comment (petele) [inline src/content/en/fundamentals/performance/audit/index.md]: indentation should match please

Comment (samdutton) [reply src/content/en/fundamentals/performance/audit/index.md]: Do you mean I should indent all follow-on lines that I've had to manually wrap to fit 80 characters?

File: src/content/en/fundamentals/performance/audit/index.md (hunk 1 above)
Comment (samdutton) [reply src/content/en/fundamentals/performance/audit/index.md]: Won't this break the markup if I wrap at 80 characters?

File: src/content/en/fundamentals/performance/audit/index.md (hunk 4 above)
Comment (samdutton) [reply src/content/en/fundamentals/performance/audit/index.md]: Fixed.

File: src/content/en/fundamentals/performance/audit/index.md (hunk 1 above)
Comment (petele) [reply src/content/en/fundamentals/performance/audit/index.md]: Nope, works fine.

File: src/content/en/fundamentals/performance/audit/index.md (hunk 5 above)
Comment (petele) [reply src/content/en/fundamentals/performance/audit/index.md]: Yes, correct.

Comment (samdutton) [reply src/content/en/fundamentals/performance/audit/index.md]: OK — done.

File: src/content/en/fundamentals/performance/audit/index.md (hunk 3 above)
Comment (samdutton) [reply src/content/en/fundamentals/performance/audit/index.md]: Done.

File: src/content/en/fundamentals/performance/audit/index.md (hunk 2 above)
Comment (samdutton) [reply src/content/en/fundamentals/performance/audit/index.md]: Done.

File: src/content/en/fundamentals/performance/audit/index.md (hunk 1 above)
Comment (samdutton) [reply src/content/en/fundamentals/performance/audit/index.md]: Changed.

Comment (samdutton) [reply src/content/en/fundamentals/performance/audit/index.md]: Adding line breaks to link markup seems to not work:

image

Is there some other way to do this?

Comment (petele) [reply src/content/en/fundamentals/performance/audit/index.md]: Looks like you figured it out.

**Pull request merged**